includes/cf/audio-info.md

```diff
@@ -1,9 +1,26 @@
-!!! info "Add the audio custom formats only if"
+??? info "I've a AppleTV - [Click to show/hide]"
 
-    - You have an audio setup that supports **ALL** HD Audio (TrueHD, DTS-X, DTS-HD, etc). *If you have a setup that only supports a few audio formats or doesn't support certain audio formats (or using only your TV speakers), it's recommended not to add any audio custom formats; else, you will limit yourself too much in what you're able to grab. Audio transcoding has a low impact on your server. If your server can't handle audio transcoding, you might consider choosing another quality profile*.
-    - You have a media player (hardware devices) that can passthrough HD audio.
-    - You don't have a AppleTV (Being it can't passthrough untouched audio).
+    ![!Audio Formats Flowchart](/Radarr/images/plex-audio-atv.png)
 
-??? question "Which audio formats do I choose?"
+    - Passthrough: The preferred method if you have an audio setup (AVR/Soundbar) so it can send the audio signals without any alteration or processing.
+    - Direct Play: The client supports the container, video stream, and audio stream natively. The Plex server simply sends the media file as-is to the client. This uses very little CPU power.
+    - Transcode: The client does not support the video stream and/or the audio stream. The Plex server re-encodes the video, audio, or both into a compatible format. Transcoding video uses a lot of CPU power, but transcoding audio uses little to moderate CPU power.
+    - Decodes: Decompresses the audio before sending it to your AVR/Soundbar.
 
-    ![!Audio Flowchart](/Radarr/images/flowchart-audio.png)
+    *partial used source: [Infuse FAQ](https://support.firecore.com/hc/en-us/articles/217735707-Audio-Options-tvOS#h_01HE1Z5XNJZK5YTF1SVTPS0MTR){:target="_blank" rel="noopener noreferrer"}*
+
+??? info "Am I losing any quality by using LPCM? - [Click to show/hide]"
+
+    - No. Since LPCM is a lossless format, using it will result in no loss of quality. What your ears hear will be exactly the same. The only difference is your receiver will recognize the audio stream as PCM instead of Dolby/DTS.
+    - But object metadata/spatial metadata is dropped. (TrueHD Atmos, DTS-X)
+
+    *source: [Infuse FAQ](https://support.firecore.com/hc/en-us/articles/217735707-Audio-Options-tvOS#h_01HE1Z5XNJZK5YTF1SVTPS0MTR){:target="_blank" rel="noopener noreferrer"}*
+
+??? info "Should I block certain audio formats because something in my setup doesn't support it? - [Click to show/hide]"
+
+    If you have chosen a profile that includes Audio Formats, it's somewhat pointless to lower the scores or block certain audio formats, because 95% of the Remuxes and UHD Encodes are provided with those HD audio formats like TrueHD Atmos, TrueHD, DTS-X, etc.
```

Comment:
```suggestion
    If you have chosen a profile that includes Audio Formats, it's somewhat pointless to lower the scores or block certain audio formats since 95% of the Remuxes and UHD Encodes provide HD audio formats such as TrueHD Atmos, TrueHD, and DTS-X.